src/compiler/compile/compiler_warnings.ts

```diff
@@ -76,6 +76,10 @@ export default {
 		code: 'a11y-unknown-role',
 		message: `A11y: Unknown role '${role}'` + (suggestion ? ` (did you mean '${suggestion}'?)` : '')
 	}),
+	a11y_no_redundant_roles: (role: string | boolean , suggestion?: string) => ({
+		code: 'a11y-no-redundant-roles',
+		message: `A11y: Redundant role '${role}'` + (suggestion ? ` (did you mean '${suggestion})?'` : '')
+	}),
```

Comment:
I doesn't seem like the `suggestion` param is used. I think we can removed that.

Reply:
Yep. I Initially imagined maybe it would suggest an alternative but the number of different scenarios a reundant role can appear in might be quite large and require different advice.